prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1522,6 +1522,8 @@ void rxr_ep_progress_internal(struct rxr_ep *ep)
 			if (ep->tx_pending == ep->max_outstanding_tx)
 				goto out;
 			ret = rxr_pkt_post_data(ep, tx_entry);
+			if (ret == -FI_EAGAIN)
```

Comment:
I see why this change is here but I'm not sure it's safe to do. We won't hit the `ep->tx_pending == ep->max_outstanding_tx` case since the loop is checking for that, but we could hit `peer->flags & RXR_PEER_IN_BACKOFF`.

If we don't set `FI_MORE` for the last packet then we're not going to ring the doorbell which could cause delayed packets or a hang.